pkg/reconciler/apis/apibinding/apibinding_reconcile_test.go

```diff
@@ -632,120 +622,6 @@ func TestReconcileBinding(t *testing.T) {
 	}
 }
 
-func TestReconcileBound(t *testing.T) {
```

Comment:
I don't think we want to remove this because there are still valid scenarios to test

Reply:
re-introduced but removed one test that asserted rebinding upon api export changes which is not possible any more.